Frontend/ui-customer/src/MerchantConfigs/Types.purs

```diff
@@ -232,4 +240,35 @@ type PurpleRideConfig = {
   visualImpairmentVideo :: String,
   physicalImpairmentVideo :: String,
   hearingImpairmentVideo :: String
+}
+type HomescreenUIConfig = {
+  primaryColor :: String,
+  pickUpViewColor :: String,
+  headerMenuButtonColor :: String,
+  headerMenuButtonImageUrl :: String,
```

Comment:
config is not required of this as of now. justt changing the image will be fine.

Reply:
It Is required. In actual design we have white icon. For sdk we need black

Reply:
> changing the image will be fine
I meant change the merchant specific image with black